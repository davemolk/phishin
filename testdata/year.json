{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":[{"id":135,"date":"1994-04-04","duration":9601071,"incomplete":false,"sbd":true,"remastered":false,"tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null}],"tour_id":22,"venue":{"id":266,"slug":"the-flynn-theatre","name":"The Flynn Theatre","other_names":[],"latitude":44.475883,"longitude":-73.212072,"shows_count":4,"location":"Burlington, VT","updated_at":"2013-10-10T02:53:56Z"},"venue_name":"The Flynn Theatre","taper_notes":"Phish - 4/4/94\r\nFlynn Theater","likes_count":28,"tracks":[{"id":2553,"show_id":135,"show_date":"1994-04-04","venue_name":"The Flynn Theatre","venue_location":"Burlington, VT","title":"Divided Sky","position":1,"duration":811964,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":2,"slug":"divided-sky","tags":[],"mp3":"https://phish.in/audio/000/002/553/2553.mp3","waveform_image":"https://phish.in/audio/000/002/553/waveform-2553.png","song_ids":[206],"updated_at":"2023-10-27T22:29:58Z"},{"id":2554,"show_id":135,"show_date":"1994-04-04","venue_name":"The Flynn Theatre","venue_location":"Burlington, VT","title":"Sample in a Jar","position":2,"duration":299781,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":1,"slug":"sample-in-a-jar","tags":[],"mp3":"https://phish.in/audio/000/002/554/2554.mp3","waveform_image":"https://phish.in/audio/000/002/554/waveform-2554.png","song_ids":[658],"updated_at":"2023-10-27T22:29:58Z"}],"updated_at":"2018-12-21T08:10:14Z"}],"updated_at":"2018-12-21T08:10:14Z"}
