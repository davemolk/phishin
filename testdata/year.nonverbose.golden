Date:       Venue:             Location:       Duration:
1994-04-04  The Flynn Theatre  Burlington, VT  2h 40m
